paradise_papers_django/search/models.py

```diff
@@ -15,7 +11,7 @@ class Entity(DjangoNode):
     service_provider = StringProperty()
     countries = StringProperty()
     jurisdiction_description =  StringProperty()
-    valid_unti = StringProperty()
+    valid_until = StringProperty()
     ibcRUC = StringProperty()
```

Comment:
Can we have the same convention on all properties? 🤔

Reply:
those are properties from the node on the database, they have to match.

![image](https://user-images.githubusercontent.com/29434543/34533512-eb3f3f80-f090-11e7-8675-444f41b221b0.png)

Reply:
Ups sorry forgot that 😅